dart.yaml

```diff
@@ -71,6 +58,17 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 179da3ba67ead156a3ba718bd798cb7a1728e741
 
+subpackages:
+  - name: ${{package.name}}-runtime
+    dependencies:
+      runtime:
+        - ld-linux
+        - glibc
+        - wolfi-baselayout
+    pipeline:
```

Comment:
no ca-certificates-bundle? no cryptography / openssl?

i wonder if packaging one of the tests apps you mentioned as a subpackage here would be useful, to have a test pipeline here in runtime to run one of them.

Reply:
missed ca-certificates-bundle, will look if openssl is required here in the morning and will add a test too.

Reply:
I think above dependencies including above should suffice for x64 and aarch64. 

There's an issue about including libatomic as well (ref: https://github.com/dart-lang/dart-docker/issues/141) but it's only required for risc-v arch. 

from upstream reference I find following info about dart-runtime: 
- https://dart.dev/tools/dart-compile#exe (self contained executables that depend on dart-runtime)
- https://dart.dev/overview#runtime (provides gc, type checks at runtime) _[I was not able to fully understood this part please let me know if looking at the description it requires more shared libraries at runtime, I think above should suffice for gc and type-checks. [Isolates](https://dart.dev/language/isolates) looks more like goroutine from a quick skim so I think nothing extra required there.]_ 

> no cryptography / openssl?

I think it's similar to Go. 
ref: https://github.com/dart-lang/crypto (A set of cryptographic functions implemented in pure Dart.)

Reply:
If needed, can be added later. note our static image (no deps of anything) still ships ca-certificates-bundle as it's difficult to add them if one copies in a static go binary, precisely to support static apps that will arrive later.